### PR TITLE
feat: added new interface to the proxy (part 1)

### PR DESCRIPTION
**Pull request by MaximeZmt**
Thank you for opening a pull request with this project, please also:

* [ ] add a brief description of your changes here
* [ ] assign the PR to yourself, or to the person(s) working on it
* [ ] start in `draft` mode and `in progress` pipeline in the project (if applicable)
* [ ] if applicable, add this PR to its related issue by one of the special keywords [Closing keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue)
* once it's ready 
  * [ ] put it in the `Review` or `Ready4Merge` pipeline in the project (if applicable) 
  * [ ] remove `draft`
  * [ ] assign a reviewer

### CODE REVIEW

File: proxy/types/signed.go [hunk 1]
```diff
@@ -25,6 +26,8 @@ func NewSignedRequest(r io.Reader) (SignedRequest, error) {
 		return req, xerrors.Errorf("failed to decode signed request: %v", err)
 	}
 
+	println(fmt.Sprintf("helloReq: %v", req))
```
Comment (PascalinDe) [inline proxy/types/signed.go]: is this debug printing?

File: scripts/run_docker.sh [hunk 2]
```diff
@@ -70,7 +70,7 @@ function init_dela() {
 function local_admin() {
   echo "adding local user $REACT_APP_SCIPER_ADMIN to admins";
   docker compose exec backend npx cli addAdmin --sciper "$REACT_APP_SCIPER_ADMIN";
-  docker compose exec backend npx cli addAdmin --sciper 987654;
+  docker compose exec backend npx cli addAdmin --sciper 654321;
```
Comment (PascalinDe) [inline scripts/run_docker.sh]: why change this?

Comment (MaximeZmt) [reply scripts/run_docker.sh]: was done for testing purpose, I've just revert to the old one👍

File: proxy/types/signed.go (hunk 1 above)
Comment (MaximeZmt) [reply proxy/types/signed.go]: yes, just removed 👍

**Review by PascalinDe — APPROVED** [review]

**Pull request merged**